modules/javafx.graphics/src/main/java/javafx/scene/text/Text.java

```diff
@@ -1267,6 +1269,42 @@ private boolean doComputeContains(double localX, double localY) {
         return TransformedShape.translatedShape(shape, x, y);
     }
 
+    /**
+     * The size of a tab stop in spaces.
+     *
+     * @return the {@code tabSize} property
+     *
+     * @defaultValue {@code 8}
+     *
+     * @since 14
+     */
+    public final IntegerProperty tabSizeProperty() {
+        return getTextAttribute().tabSizeProperty();
+    }
+
+    /**
+     * Gets the size of a tab stop in spaces.
+     * @return the size of a tab in spaces
+     * @since 14
+     */
```

Comment:
The javadoc property support only requires one of the setXxxx/getXxxx/xxxxProperty methods to have a javadoc comment block. The javadoc tool takes care of documenting the property itself and all three methods. See `wrappingWidth` for a good example.